src/features/locators/types/locator.types.ts

```diff
@@ -72,28 +89,16 @@ export interface Locator extends PredictedEntity {
   active?: boolean;
   isCustomName?: boolean;
   isCustomLocator?: boolean;
+  isCreatedByUser?: boolean; // we need it to locator icon conditional render
   locatorType?: LocatorType;
   pageObj: PageObjectId;
   parent_id: JDNHash;
   priority?: LocatorCalculationPriority;
   type: ElementClass;
-  validity?: Validity;
+  message: LocatorValidationErrorType;
```

Comment:
as I see from code, it can be a string as well

Reply:
yes, type LocatorValidationErrorType contains a string